tests/components/application-connector/test/compass-runtime-agent/synchronisation_test.go

```diff
@@ -32,18 +36,22 @@ func (cs *CompassRuntimeAgentSuite) TestCreatingApplications() {
 	cs.Assert().NoError(err)
 
 	// Compare Application created by Compass Runtime Agent with expected result
-	err = cs.appComparator.Compare(expectedAppName, synchronizedCompassAppName)
-	cs.Assert().NoError(err)
 
-	// Clean up
-	err = cs.directorClient.UnassignApplication(applicationID, cs.formationName)
-	cs.Assert().NoError(err)
+	cs.Run("Is app correct", func() {
```

Comment:
I'd change the test names to some `Compass Runtime Agent should create Application` and `Compass Runtime Agent should remove Application`

Reply:
Technically the app is created few lines above, this only verifies if the created app is correct